### PR TITLE
Fixes #1977

### LINKED ISSUE #1977: [STALE] docs/learn-staking.md
Last edit happened 45 days ago 😬

### PR DESCRIPTION
**Pull request by Swader**

### CODE REVIEW

File: docs/learn-staking.md [hunk 1]
```diff
@@ -32,23 +30,20 @@ validator set.
...
 
 ### 2. Nomination period
 
 Any potential validators can indicate their intention to be a validator candidate. Their candidacies
 are made public to all nominators, and a nominator in turn submits a list of any number of
-candidates that it supports. In the next epoch (lasting several hours), a certain number of
-validators having the most DOT backing get elected and become active. There are no particular
-requirements for a DOT holder to become a nominator, though we expect each nominator to carefully
-track the performance and reputation of validators.
+candidates that it supports. In the next epoch (lasting 4 hours on Polkadot, 1 hour on Kusama), a
```
Comment (jakehemmerle) [inline docs/learn-staking.md:42]: nitpick: In the next epoch (each epoch lasts 4 hours on Polkadot, 1 hour on Kusama),

File: docs/learn-staking.md [hunk 2]
```diff
@@ -62,41 +57,49 @@ and achieve fair representation of the nominators. If you want to know more abou
...
+mechanism will split its stakes into pieces `s_1`, `s_2`, …, `s_k`, so that it backs validator `i`
+with stake `s_i`. In that case, nominator `n` will be rewarded essentially the same as if there were
+`k` nominators in different pools, each backing a single validator `i` with stake `s_i`.
+
+For each validator pool, we keep a list of nominators with the associated stakes.
 
 The general rule for rewards across validator pools is that two validator pools get paid essentially
-the **same amount of DOT** for equal work, i.e. they are NOT paid proportional to the stakes in each
-pool. There is a probabilistic component to staking rewards in the form of
+the **same amount of tokens** for equal work, i.e. they are NOT paid proportional to the stakes in
```
Comment (jakehemmerle) [inline docs/learn-staking.md:70]: should we swap DOT for tokens in most other places in this article?

File: docs/learn-staking.md [hunk 3]
```diff
@@ -182,23 +185,19 @@ Polkadot and the cryptography behind it [see here](learn-keys).
...
-it," while the validator will be doing active service for the network by performing the critical
-operations. For this reason, the validator has certain privileges regarding the payout of the
-staking mechanism and will be able to declare its own allocation before the share is divided to
-nominators.
+Since validator slots are limited, most of those who wish to stake their DOT and contribute economic
+security to the network will be nominators.
 
-![staking](assets/NPoS/article-2.png)
+Validators do most of the heavy lifting: they produce new block candidates in BABE, vote and come to
+consensus in GRANDPA, validate the state transition function of parachains, and possibly some other
```
Comment (jakehemmerle) [inline docs/learn-staking.md:192]: Since this is on a 'learn' page would it be worth including SASSAFRAS somewhere?

File: docs/learn-staking.md [hunk 4]
```diff
@@ -415,7 +414,8 @@ stake acts just as if they were another nominator.
 
 Rewards can be directed to the same account (controller), to the stash account (and either
 increasing the staked value or not increasing the staked value), or to a completely custom account.
-It is also possible to top-up / withdraw some bonded DOT without having to un-stake everything.
+It is also possible to top-up / withdraw some bonded DOT without having to un-stake everything. You
+can also send rewards to "None", effectively burning them.
```
Comment (jakehemmerle) [inline docs/learn-staking.md]: I don't see a "None" option under "payment destination" or "payment destination" > "the payment account"

Comment (jakehemmerle) [reply docs/learn-staking.md]: we can just remove the last line, or maybe replace a zero/burn/treasury address with 'None' here if an address like that exist

File: docs/learn-staking.md (hunk 2 above)
Comment (Swader) [reply docs/learn-staking.md:70]: No need to do so in examples, but in references to tokens, sure. Making it more mirroring-friendly basically.

File: docs/learn-staking.md (hunk 3 above)
Comment (Swader) [reply docs/learn-staking.md:192]: No, it's not live yet or being advertised, so no need to mention it yet

File: docs/learn-staking.md (hunk 4 above)
Comment (Swader) [reply docs/learn-staking.md]: Should definitely be there, I'll need to check

File: docs/learn-staking.md (hunk 1 above)
Comment (Swader) [reply docs/learn-staking.md:42]: Nyeh :D

**Review by emresurmeli — APPROVED** [review]

File: docs/learn-staking.md (hunk 4 above)
Comment (Swader) [reply docs/learn-staking.md]: It's in extrinsics

**Review by jakehemmerle — COMMENTED** [review]
lgtm

**Pull request merged**